spec/requests/resources_spec.rb

```diff
@@ -31,6 +31,23 @@
         expect(returned_ids).to match_array(resources_a.map(&:id))
       end
     end
+
+    context 'with a category_id and latitude/longitude' do
+      close = 10, far = 100, further = 200
+      let!(:category) { create :category, name: 'a' }
+      let!(:resources) do
+        create_list :resource, 3, categories: [category]
+      end
+      let!(:address_further) { create :address, latitude: further, longitude: further, resource_id: resources[0].id }
+      let!(:address_close) { create :address, latitude: close, longitude: close, resource_id: resources[1].id }
+      let!(:address_far) { create :address, latitude: far, longitude: far, resource_id: resources[2].id }
+      it 'returns only resources with that category' do
```

Comment:
I think we're interested in the order of the results in this test rather than that they're in the correct category